app/Console/Commands/RepoStats.php

```diff
@@ -57,6 +58,7 @@ public function handle()
         ($dateFrom && $dateTo)
             ? $gitLogRunner->setDateRange($dateFrom, $dateTo)
             : $gitLogRunner->setDateRangeLastMonth();
+        $metrics = $this->option('metrics') ?: 'direct,tracked,self-merges,pull-requests';
```

Comment:
the default value should bring all metrics, so that we don't maintain this piece of code when we add new metrics.  and we always (until for now) display all. I think.